hook!-full-game/src/Player/Hook/SnapDetector.gd

```diff
@@ -1,19 +1,15 @@
-tool
```

Comment:
That's why you're getting the error, even to get called in the editor a node has to use the tool mode.